HW2- SQL File

```diff
@@ -0,0 +1,78 @@
+/*-- BDA 696 HW 2 --
+ * Karenina Zaballa
+
+
+INSTRUCTIONS:
```

Comment:
Comment this out. you added the */ on the bottom but forgot the /* at the top before instructions

Reply:
Not sure if you saw, but the top line where I had the class title, I have the beginning "/*". Sharp eye though